lib/passageidentity/client.rb

```diff
@@ -22,7 +22,10 @@ module Passage
                :user_metadata_schema,
                :layouts,
                :default_language,
+               :auth_fallback_method,
+               :auth_fallback_method_ttl,
```

Comment:
These fields should also be listed in the SDK README [here](https://github.com/passageidentity/passage-ruby/blob/ff443b69188b699203798e09eb485ceb7a759907/README.md#retrieve-app-info).